exercises/main.tf.completed

```diff
@@ -39,7 +39,7 @@ resource "google_compute_firewall" "http-server" {
 }
 
 resource "tls_private_key" "ssh-key" {
-  algorithm = "RSA"
+  algorithm = "ED25519"
   rsa_bits  = "4096"

```

Comment:
If we're switching this to an Ed25519 key, we should remove the `rsa_bits` parameter; it's not applicable to Ed25519 keys, and I'm not sure if we keep it that TF won't complain. Probably good to remove it either way for clarity's sake.